docs/reference/configuration/keywords.rst

```diff
@@ -859,14 +859,51 @@ and/or history matching project.
         model, it is not possible for ERT to fully validate the ``GEN_DATA`` keyword
         at configure time. If for instance your forward model generates a file
         ``SimulatedWOC_0`` (without the ``.txt`` extension you have configured), the
-        configuration problem will not be detected before ERT eventuallly fails to load
+        configuration problem will not be detected before ERT eventually fails to load
         the file ``SimulatedWOC_0.txt``.
 
 
 .. _gen_kw:
 .. topic:: GEN_KW
 
         The General Keyword, or :code:`GEN_KW` is meant used for specifying a limited number of parameters.
+        :code:`GEN_KW` supports either 2 or 4 positional arguments, as well as a few keyword arguments. The first
+        parameter is always the name of the parameter group. If given two positional arguments, those are:
+
+        ::
+
+                GEN_KW  <name of parameter>  <prior distribution file>
+
+
+        where:
+
+        | :code:`<name of parameter>` is an arbitrary unique identifier
+        | :code:`<prior distribution file>` is a file containing :ref:`parameter definitions <prior_distributions>`
+
+
+        In the case of 4 positional arguments, those are:
+
+        ::
+
+                GEN_KW  <name of parameter>  <template file> <output file on runpath> <prior distribution file>
+
+        where:
+
+        | :code:`<name of parameter>` is an arbitrary unique identifier
+        | :code:`<template file>` is an input :ref:`template file <gen_kw_template_file>`,
+        | :code:`<output file on runpath>` name of the output file from ert containing templated values,
+        | :code:`<prior distribution file>` is a file containing :ref:`parameter definitions <prior_distributions>`
+
+        Keyword arguments:
+
+        ::
+
+                GEN_KW  ... UPDATE:TRUE/FALSE INIT_FILES:path/to/file_%d
+
+        where :code:`UPDATE` means if this parameter should be included in a history match run,
```

Comment:
I suggest the following rephrasing that is perhaps a bit clearer:

Where the :code:`UPDATE` keyword argument specifies whether a parameter group should be included during the history matching process.
It must be set to either TRUE or FALSE.